units/src/value.rs

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Bitcoin has value, and that value is denominated in sats - period.
```

Comment:
Or, if this is too provocative:

```suggestion
//! A UTXO has value, and that value is denominated in satsoshis.
```